File: songshizz_bot/Songwhip/Models/HasLinks.cs
using System.Text.Json.Serialization;

namespace Songwhip.Models;

public class HasLinks
{
    [JsonPropertyName("tidal")]
    public bool Tidal { get; set; }

    [JsonPropertyName("amazon")]
    public bool Amazon { get; set; }

    [JsonPropertyName("deezer")]
    public bool Deezer { get; set; }

    [JsonPropertyName("itunes")]
    public bool Itunes { get; set; }

    [JsonPropertyName("napster")]
    public bool Napster { get; set; }

    [JsonPropertyName("pandora")]
    public bool Pandora { get; set; }

    [JsonPropertyName("spotify")]
    public bool Spotify { get; set; }

    [JsonPropertyName("youtube")]
    public bool Youtube { get; set; }

    [JsonPropertyName("amazonMusic")]
    public bool AmazonMusic { get; set; }

    [JsonPropertyName("itunesStore")]
    public bool ItunesStore { get; set; }

    [JsonPropertyName("youtubeMusic")]
    public bool YoutubeMusic { get; set; }
}

File: modules/songwhip_api/models/has_links.go
package models

type HasLinks struct {
	Tidal        bool `json:"tidal"`
	Amazon       bool `json:"amazon"`
	Deezer       bool `json:"deezer"`
	Itunes       bool `json:"itunes"`
	Napster      bool `json:"napster"`
	Pandora      bool `json:"pandora"`
	Spotify      bool `json:"spotify"`
	Youtube      bool `json:"youtube"`
	AmazonMusic  bool `json:"amazonMusic"`
	ItunesStore  bool `json:"itunesStore"`
	YoutubeMusic bool `json:"youtubeMusic"`
}
